### DISCUSSION TITLE
Animation tutorial/example

### DISCUSSION THREAD

**Discussion by justinstroud-campmeeker** [MrClock8163/Arma3ObjectBuilder#69] [Q&A]
A basic character animation example, or a (super-basic) tutorial. Something very simple, like waving "hello" or even a small calibration. Nothing fancy or time consuming.

A general blender-to-arma video (or text, doesn't matter) that does some stupid, minor animation.

Personally, I've been able to get the RTM exported with an animation using an armature that is 1-1 mapped to the standard "man" skeleton. I have some major issues with scaling and pivots but I don't know the steps needed to fix it (or if I took the right steps to begin with.)

**Comment by MrClock8163** [comment]
Your results obviously largely depend on the rig you're using. The best public rig for the `OFP2_ManSkeleton` can be downloaded from ModDB through the [arma forums](https://forums.bohemia.net/forums/topic/161228-armarig-for-blender/).

Personally I only ever do RTM animations for weapon hand placements, which are not really actual animations. I did one or two reload animations too, but that's about it, so I'm not very experienced.

For the animation configurations I suggest you look at the configs in mods that have working examples.

**Comment by ShadySauce26** [comment]
everytime i try to export my animations and load up arma 3 the hands get deformed and streched. even tried importing a vanilla anim.rtm file and re exporting it with the same result

**Comment by MrClock8163** [comment]
If you believe it to be a bug, please submit a bug report issue with detailed descriptions, steps to reproduce and lots of images.

**Comment by ShadySauce26** [comment]
<img width="1435" height="821" alt="Screenshot 2025-09-17 145739" src="https://github.com/user-attachments/assets/42724283-03c2-4d8e-8852-46d3c6c4a9d8" />
<img width="1012" height="569" alt="Screenshot 2025-09-17 150232" src="https://github.com/user-attachments/assets/928e00bc-9be8-4bd0-8ff3-432eb0c65959" />
So im pretty much brand new to modding for arma 3 and i watched your video on how to set up the skeleton. ive got the OFP2 set up and im using the ManSkeleton_Pivots.p3d as the skeleton. then i used one of armas man example p3ds (cant remember the name of it) just as a visual helper. Ive been working on this pump shotgun for some time and ive got everything working, but the hand animations i was using from another mod dont line up with my model. at first i tried importing the animations i was using and adjusting the positions of everything which kind of worked but everytime the weapon was charged (pumped) the characters hand would teleport around. then i tried just getting the static hand animation using one of armas vanilla stance animations but when i tried to export it the hands and fingers stretched into something i can only describe as the creature from the thing movies. Next i tried just importing a vanilla animation and immediately re exporting it onto my character and had the same result with the hands getting streched out. last thing i tried was just setting up the skeleton, leaving it in the default t-pose, inserting a single keyframe with the start end set as 0:0, and exporting that which resulted in the image ive posted here where the characters arms are bent in the outward position. ive tried posting asking for help on multiple discord pages with no help and even tried asking chatgpt but still cannot figure out what im doing wrong

**Reply by MrClock8163** [reply]
What you show in the two screenshots is prefectly normal and expected. It's a consequence of how hand placement animations work.

The other issues you described can be caused by a bunch of mistakes.
For RTMs to work properly you need to binarize your PBOs, possibly have a model config file (depends on the packing tool), have correct casing of the bone names (again depends on the packing tool), etc.
